### PR TITLE
[ISSUE #2006] Add request-reply doc

### PR DESCRIPTION
**Pull request by keranbingaa**
## What is the purpose of the change

add a doc to explain the request-reply  feature

## Brief changelog

add a doc and some pictures

### CODE REVIEW

File: docs/cn/rpc_request.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+                    } catch (MQClientException | RemotingException | MQBrokerException | InterruptedException e) {
+                        e.printStackTrace();
+                    }
+                }
+                return ConsumeConcurrentlyStatus.CONSUME_SUCCESS;
+            }
+```
+
+## 4 接口参数
+### 4.1 public Message request(Message msg,long timeout)
```
Comment (RongtongJin) [inline docs/cn/rpc_request.md:93]: Interfaces should not be used as level 3 titles

**Comment by RongtongJin** [comment]
plz submit related issues to link this PR

**Comment by keranbingaa** [comment]
> plz submit related issues to link this PR

ok, I submit a issue #2006

File: docs/cn/rpc_request.md (hunk 1 above)
Comment (keranbingaa) [reply docs/cn/rpc_request.md]: get it, I modify the interfaces to plain text

File: docs/cn/rpc_request.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+```
+Message msg = new Message(topic,
+                "",
+                "Hello world".getBytes(RemotingHelper.DEFAULT_CHARSET));
+
+            long begin = System.currentTimeMillis();
+            Message retMsg = producer.request(msg, ttl);
+            long cost = System.currentTimeMillis() - begin;
+            System.out.printf("request to <%s> cost: %d replyMessage: %s %n", topic, cost, retMsg);
+```
```
Comment (RongtongJin) [inline docs/cn/rpc_request.md:63]: It would be better if the code was aligned

File: docs/cn/rpc_request.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+                        // create reply message with given util, do not create reply message by yourself
+                        Message replyMessage = MessageUtil.createReplyMessage(msg, replyContent);
+
+                        // send reply message with producer
+                        SendResult replyResult = replyProducer.send(replyMessage, 3000);
+                        System.out.printf("reply to %s , %s %n", replyTo, replyResult.toString());
+                    } catch (MQClientException | RemotingException | MQBrokerException | InterruptedException e) {
+                        e.printStackTrace();
+                    }
+                }
```
Comment (RongtongJin) [inline docs/cn/rpc_request.md:87]: It would be better if the code was aligned

**Pull request merged**